.github/workflows/docker-tests.yml

```
@@ -1,21 +1,50 @@
-name: CentOS7-tests@TF
+name: docker-tests at Testing Farm
 
 on:
   issue_comment:
     types:
       - created
 jobs:
   build:
+    strategy:
```

```suggestion
    strategy:
        fail-fast: false
```
This "fail-fast: false" option should ensure, that all test options in the matrix would be run, even though one of them fails (the default is the opposite). Please ignore the comment, if this behavior is secured somehow else in this Github Action.